api/src/main/java/org/openmrs/scheduler/SchedulerConstants.java

```diff
@@ -12,25 +12,25 @@
 public class SchedulerConstants {
 	
 	// Number of milliseconds per second (used for readability)
-	public static int SCHEDULER_MILLIS_PER_SECOND = 1000;
+	public static final int SCHEDULER_MILLIS_PER_SECOND = 1000;
 	
 	// 0 second delay added before the initial start of a task
-	public static long SCHEDULER_DEFAULT_DELAY = 0;
+	public static final long SCHEDULER_DEFAULT_DELAY = 0;
 	
-	public static String SCHEDULER_DEFAULT_USERNAME = "admin";
+	public static final String SCHEDULER_DEFAULT_USERNAME = "admin";
 	
-	public static String SCHEDULER_DEFAULT_PASSWORD = "test";
+	public static final String SCHEDULER_DEFAULT_PASSWORD = "test";
```

Comment:
I think you might want to remove the code in SchedulerUtil.startup() that assigns values to these fields.

Reply:
@wluyima on it :)

Reply:
@wluyima I removed that code and pushed..Thanks